src/views/Users/UserDetails.vue

```diff
@@ -33,8 +33,8 @@
             <base-card v-if="isResourceTableVisible">
               <resource-availability-table
                 :tableColumns="tableColumnsResources"
-                :resourceItem="resourceItemResources"
-                :user="user"
+                :resourceItem="purchasedResources"
+                :orgName="user.firstName"
```

Comment:
Okay, this answers my question about the ResourceAvailabilityTable component. This table should be a generic component that can be used either for a user showing their purchased resources or by an organization to show their available resources and we should have an option to transfer/remove those resources from the table when it's shown in the organization's view. Therefore the buttons should be added as children components somehow. And the orgName property should just be name in this case as it might be used to mean both user's resources or organization's resources.